scriptmodules/emulators/reicast.sh

```diff
@@ -42,7 +42,15 @@ function configure_reicast() {
 
     # create bios dir. Copy dc_boot.bin and dc_flash.bin there.
     mkdir $md_inst/data
+
+    cat > $md_inst/reicast.sh << _EOF_
+#!/bin/bash
+cd /opt/retropie/emulators/reicast
+sudo ./reicast.elf -config config:image="$1"
```

Comment:
@joolswills how do i write "$1" to a file. I don't want to write the value of variable $1 of module reicast into reicast.sh.